src/command_modules/azure-cli-acr/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "2.2.8"
+VERSION = "2.2.9"
```

Comment:
Is 2.2.8 released though? If not (https://pypi.org/project/azure-cli-acr/), you want to keep 2.2.8 and merge history files instead.

Reply:
I see, 2.2.8 is not released yet. When is the next release? Will it be able to include this PR?

Reply:
I'm guessing there will be no update on this section of the code when we add features to the ACR module will az version be consistent. Should we add a test to ensure this?

Reply:
You say, add a test to ensure that the version output for `az acr check-health` is the same outputted for `az --version`?

Reply:
Test is not required if are calling a core function.

Reply:
So you think that's ok?

Reply:
Release history: https://github.com/Azure/azure-cli/releases
I think we can merge this with `2.2.8` if we are not going to sit on this for long.